src/server_name.rs

```diff
@@ -49,6 +51,18 @@ impl<'a> ServerName<'a> {
             Self::IpAddress(i) => ServerName::IpAddress(*i),
         }
     }
+
+    /// Return the string representation of this `ServerName`.
+    #[cfg(all(feature = "alloc", feature = "std"))]
```

Comment:
`std` implies `alloc` so we should use `feature = "std"` in favor of `all(feature = "alloc", feature = "std")`.

Reply:
Fixed throughout.